pkg/operator/controller/names.go

```diff
@@ -106,6 +106,25 @@ func RouterOperatorGeneratedDefaultCertificateSecretName(ci *operatorv1.IngressC
 	}
 }
 
+// ClientCAConfigMapName returns the namespaced name for the client CA
+// configmap.
+func ClientCAConfigMapName(ic *operatorv1.IngressController) types.NamespacedName {
+	return types.NamespacedName{
+		Namespace: "openshift-ingress",
+		// TODO Is this user-specified or operator-managed?
```

Comment:
Is this an open question, or is this a TODO saying more logic needs to be added here?

Reply:
I'm not sure what I was thinking when I wrote this comment, but I realize now, the configmap should be in the `openshift-config` namespace, and I should add a controller to copy the configmap from the `openshift-config` namespace to the `openshift-ingress` namespace.

Reply:
Right, that makes sense.